src/helm/helmfile.yaml

```diff
@@ -65,3 +68,61 @@ releases:
       - env.d/{{ .Environment.Name }}/values.secrets.yaml
     secrets:
       - env.d/{{ .Environment.Name }}/secrets.enc.yaml
+
+  - name: dev-backend
+    namespace: {{ .Namespace }}
+    chart: dev-backends/dev-backend
+    version: 0.0.1
+    values:
+      - postgres:
+          enabled: true
+          name: postgres
+          #serviceNameOverride: postgres
+          image: postgres:16-alpine
+          username: dinum
+          password: pass
+          database: dinum
+          size: 1Gi
+      - redis:
+          enabled: true
+          name: redis
+          #serviceNameOverride: redis
+          image: redis:8.2-alpine
+          username: user
+          password: pass
+      - minio:
+          enabled: true
+          image: minio/minio
+          name: minio
+          #serviceNameOverride: minio
+          hostname: minio.127.0.0.1.nip.io
+          tls:
+            enabled: true
+            secretName: meet-tls
+          username: dinum
+          password: password
+          bucket: meet-media-storage
+          size: 1Gi
+      - keycloak:
+          enabled: true
+          image: quay.io/keycloak/keycloak:20.0.1
+          name: keycloak
+          #serviceNameOverride: keycloak
+          hostname: keycloak.127.0.0.1.nip.io
```

Comment:
You should add a suffix here to not collapse with other projects running in a different namespace.

Also, we should find a solution to have one keycloak for all our projects in the dev stack ? Maybe running it in a dedicated namespace ? And we use a convention to use this namespace accross all our projects ? We will need this once we will implement the resource server protocol on our projects.